### ISSUE TITLE
[White Label -AudioCard]: Narrator announcing unnecessary list information when focus lands on whole audio file while navigating using arrow keys.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5722]
### What is the PWD impact?

Screen reader users encounter confusing and irrelevant announcements, disrupting the logical reading order and making it harder to navigate and understand the page content.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Focus management

### How to reproduce the issue?

Pre-Requisite:
Turn on Narrator

Repro Steps
1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type audiocard word in type your message text box.
4. Response will appear.
5. Navigate to the bot response using arrow keys.
6. Observe an issue that Narrator announcing unnecessary list information when focus lands on whole audio file while navigating using arrow keys.

### What do you expect?

In scan mode, the screen reader focus should only move to visible and meaningful elements that convey relevant content or interactivity to users. Hidden elements should be skipped.

### What actually happened?

Narrator announcing unnecessary list information when focus lands on whole audio file while navigating using arrow keys.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/3ad52b7d-6e2d-415e-98ba-5d5146fc5f5f

### Adaptive Card JSON

```json

```
